packages/subgraph/test/validation/hol/flowOperatorValidator.ts

```diff
@@ -65,6 +65,10 @@ const validateFlowOperator = (
         flowOperatorEntity.flowRateAllowanceGranted,
         "Flow Operator: flowRateAllowanceGranted error"
     ).to.equal(expectedFlowOperator.flowRateAllowanceGranted);
+    expect(
```

Comment:
please remove this, we ideally want to remove code from `packages/subgraph/test`